<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
    <head>
        <META http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>Standard Error from JUnitTclTest</title>
    </head>
    <body>
        <pre>Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'. The driver is automatically registered via the SPI and manual loading of the driver class is generally unnecessary.
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .DOPCSV.DatabaseManager
Because:
Access denied for user 'spacecadet'@'35.192.10.37' (using password: YES)
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
Caused by: java.sql.SQLException: Access denied for user 'spacecadet'@'35.192.10.37' (using password: YES)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828)
	at com.mysql.cj.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.java:448)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ptolemy.actor.lib.database.DatabaseManager$MakeConnection.run(DatabaseManager.java:633)
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .DOPCenter.DatabaseManager
Because:
Access denied for user 'spacecadet'@'35.192.10.37' (using password: YES)
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
Caused by: java.sql.SQLException: Access denied for user 'spacecadet'@'35.192.10.37' (using password: YES)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828)
	at com.mysql.cj.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.java:448)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ptolemy.actor.lib.database.DatabaseManager$MakeConnection.run(DatabaseManager.java:633)
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failed to open connection to the database.
  in .Placard.DatabaseManager
Because:
Access denied for user 'spacecadet'@'34.66.25.221' (using password: YES)
	at ptolemy.actor.lib.database.DatabaseManager.getConnection(DatabaseManager.java:566)
	at ptolemy.actor.lib.database.DatabaseManager.preinitialize(DatabaseManager.java:583)
	at ptolemy.actor.Director.preinitialize(Director.java:1302)
	at ptolemy.actor.Director.preinitialize(Director.java:1257)
	at ptolemy.domains.sdf.kernel.SDFDirector.preinitialize(SDFDirector.java:763)
	at ptolemy.actor.CompositeActor.preinitialize(CompositeActor.java:1828)
	at ptolemy.actor.Manager.preinitializeAndResolveTypes(Manager.java:1040)
	at ptolemy.actor.Manager.initialize(Manager.java:710)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1933)
Caused by: java.sql.SQLException: Access denied for user 'spacecadet'@'34.66.25.221' (using password: YES)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:828)
	at com.mysql.cj.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.java:448)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at ptolemy.actor.lib.database.DatabaseManager$MakeConnection.run(DatabaseManager.java:633)
</pre>
    </body>
</html>
